app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,47 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+section {
+  overflow: auto;
+}
+.center {
+  text-align: center;
+  vertical-align: center;
+}
+.center h1 {
+	margin-top: 180px;
+  margin-bottom: 30px;
+  color: #000000;
+}
+.jumbotron{
+	background: url(eventhero.jpg);
+	background-size: 100%;
+	height: 100%;
+	margin-bottom: 0px;
+	a{
+	margin-bottom: 150px;
+	}
+	h1{
+		font-size: 80px;
```

Comment:
Properties should be ordered color, font-family, font-size, font-style, font-weight